src/Metadata/NodeAnnotationMetadata.php

```diff
@@ -15,38 +15,16 @@
 
 final class NodeAnnotationMetadata
 {
-    /**
-     * @var string
-     */
-    private $label;
-
-    /**
-     * @var string
-     */
-    private $customRepository;
-
-    /**
-     * @param string      $label
-     * @param string|null $repository
-     */
-    public function __construct($label, $repository)
+    public function __construct(private string $label, private ?string $customRepository)
```

Comment:
Please, set the default null value, because you will get not accessable exception